src/ElmSharp/ElmSharp/Elementary.cs

```diff
@@ -49,6 +49,7 @@ public enum FocusAutoScrollMode
     public static class Elementary
     {
         private const string _themeFilePath = "/usr/share/elm-sharp/elm-sharp-theme.edj";
+        private const string _vdThemeFilePath = "/usr/share/elm-sharp/vd-elm-sharp-theme.edj";
```

Comment:
What abour `tvTheme` instead of `vdTheme`?

Reply:
Done